suzieq/engines/pandas/evpnVni.py

```diff
@@ -22,6 +22,9 @@ def get(self, **kwargs) -> pd.DataFrame:
 
         addnl_fields = []
         fields = self.schema.get_display_fields(columns)
+        self._add_active_to_fields(kwargs.get('view', 'latest'), fields,
+                                   addnl_fields)
```

Comment:
`view` is not a kwargs. It's stored inside self.iobj.view. `kwargs.get('view', 'latest')` returns always 'latest'.
My suggestion is to modify the function `_add_active_to_fields`.

Reply:
We had a discussion about this on Slack. We're going to leave it as is for now